src/SDKs/DataFactory/DataFactory.Tests/JsonSamples/TriggerJsonSamples.cs

```diff
@@ -149,6 +149,60 @@ public class TriggerJsonSamples : JsonSampleCollection<TriggerJsonSamples>
     }
   }
 }
+";
+
+        [JsonSample]
```

Comment:
nit: please fix indentation

Reply:
The current Json samples also use these indentation, 8 spaces before  [JsonSample], what do you want me to change to?